job-server-python/src/python/sparkjobserver/subprocess.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import print_function
+
+import sys
+from importlib import import_module
+from py4j.java_gateway import JavaGateway, java_import, GatewayClient
+from pyhocon import ConfigFactory
+from pyspark.context import SparkContext, SparkConf
+from pyspark.sql import SQLContext, HiveContext
+from sparkjobserver.api import ValidationProblem
+
+def exit_with_failure(message, exitCode = 1):
+    print(message, file = sys.stderr)
+    sys.exit(exitCode)
+
+def import_class(cls):
+    (module_name, class_name) = cls.rsplit('.', 1)
+    module = import_module(module_name)
+    c = getattr(module, class_name)
+    return c
+
+if __name__ == "__main__":
```

Comment:
Should we have something like pydoc or whatever is the standard for Python app documentation?